mix.exs

```diff
@@ -21,20 +20,17 @@ defmodule NewRelixir.Mixfile do
   end
 
   defp deps do
-    [{:phoenix, "~> 1.3-rc"},
-     {:ecto, ">= 1.1.0 and < 3.0.0"},
-     {:newrelic, "~> 0.1.0"},
-     {:earmark, "~> 0.1", only: :dev},
-     {:ex_doc, "~> 0.11", only: :dev}]
+    [
+      {:ecto, ">= 1.1.0 and < 3.0.0"},
+      {:ex_doc, "~> 0.11", only: :dev},
+      {:lhttpc, "~> 1.4"},
+      {:phoenix, "~> 1.3"}
+    ]
   end
 
   defp package do
     [maintainers: ["Fredrik Björk", "Robert Zotter"],
      licenses: ["MIT"],
      links: %{"GitHub" => "https://github.com/TheRealReal/new-relixir"}]
   end
-
-  defp aliases do
-    ["test": ["test --no-start"]]
```

Comment:
I think we'll still need this to have more control over supervision tree. We don't want to bring it all up on most of the unit tests, for example.